react/server.js

```diff
@@ -0,0 +1,256 @@
+// Express server previewing how to best interact with common Pelm API endpoints
+import express from 'express';
+import fetch, { Headers } from 'node-fetch';
+import dotenv from 'dotenv';
+import session from 'express-session'
+const app = express()
+const port = 3001
+dotenv.config()
+
+// Enviornment variables stored in the .env file you created
+const PELM_CLIENT_ID = process.env.PELM_CLIENT_ID
+const PELM_SECRET = process.env.PELM_SECRET
+const USER_ID = process.env.USER_ID
+
+
+app.use(
+  // Here we are using session in place of a database
+  // Instead use your own database to store values like the Authorization_Token
+  session({ secret: "secret", saveUninitialized: true, resave: true })
+);
+
+
+app.use(express.json())
+app.use(express.urlencoded({ extended: true }))
+
+
+// Create connect_token
+// Link to Pelm docs: https://docs.pelm.com/reference/post_auth-connect-token
+app.get('/connect', (req, res) => {
```

Comment:
let's name this endpoint `/connect-token`

Reply:
Changed!